modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/package.scala

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.web.client.components.sequence
+
+package object steps {
+  implicit class ControlButtonResolverSyntax[A](val a: A) extends AnyVal {
```

Comment:
I think we often name these as `ControlButtonResolverOps` but that's a tiny detail

Reply:
Ok, let's unify.